.github/workflows/build.yml

```diff
@@ -92,14 +93,17 @@ jobs:
               exit 1
           fi
           echo "IMAGE_NAME=${{ matrix.image_name }}-${IMAGE_FLAVOR}" >> $GITHUB_ENV
+          echo "BUILD_ARCH=${{ matrix.arch }}" >> $GITHUB_ENV
 
       - name: Generate tags
         id: generate-tags
         shell: bash
         run: |
           # Generate a timestamp for creating an image version history
           TIMESTAMP="$(date +%Y%m%d)"
-          VARIANT="${{ matrix.major_version }}"
+          VARIANT="${{ matrix.major_version }}"-"${{ matrix.arch }}"
```

Comment:
i wasn't sure if this was useful or if splitting the values was better. Opened the PR like this in case folks had a preference